src/data/PredictorD2ApiRepository.ts

```diff
@@ -50,7 +50,7 @@ export class PredictorD2ApiRepository implements PredictorRepository {
         return Future.join2(predictorData$, schedulingData$).map(([{ objects }, schedulingData]) => {
             return objects.map(item => {
                 const scheduling = schedulingData.find(s => s.id === item.id) ?? { sequence: 0, variable: 0 };
-                return { ...item, scheduling };
+                return { ...item, scheduling, organisationUnitDescendants: "DESCENDANTS" };
```

Comment:
So, `organisationUnitDescendants` is missing in `D2PredictorSchema` definition.
This means that even when the `organisationUnitDescendants` property exists in `item`, TS will complain that its missing.
Your fix is a hard-coded value, what about the `organisationUnitDescendants: "SELECTED"` cases?

Same for line 89.

Reply:
Ok then. Marking as resolved.